src/data/actions/courseInfo.js

```diff
@@ -93,9 +103,9 @@ function fetchCourseInfo(id) {
   };
 }
 
-function createCourseRun(course, courseRunData) {
+function createCourseRun(course, courseRunData, fromCourseCreatePage=false) {
```

Comment:
Nit: I would name this to be `sCourseCreate` or anything else that does not include `Page` or `from`.